### DISCUSSION TITLE
Should an explicit SVG transformer be necessary for Vue SFCs?

### DISCUSSION THREAD

**Discussion by msimpson** [parcel-bundler/parcel#6621] [Q&A]
I noticed an inability to load SVG images from Vue SFCs if they are specified inside the `<template>`.

The build kept failing due to a lack of transformer, which I resolved using:

```
{

  "extends": "@parcel/config-default",
  
  "transformers": {
  
    "*.svg": [ "@parcel/transformer-raw" ]
  
  }

}
```

Are you expected to provide transformers for Vue SFCs or is this more of a WIP situation?

**Comment by DeMoorJasper** [comment]
You could have a look at the docs site for how to use vue with parcel https://v2.parceljs.org/languages/vue/

**Reply by msimpson** [reply]
> Parcel supports Vue without the need for any additional configuration.
> (https://v2.parceljs.org/languages/vue/)

Parcel 2 seems to handle SVGs elsewhere without any additional configuration.

Why is Vue different and where is that mentioned?

***EDIT***

To be clear, if I use an SFC like:

```
<script>

  export default {};

</script>

<style scoped>

  div {
  
    background-image: url( ../images/image.svg );
  
  }

<style>

<template>

  <div>Example</div>

</template>
```

`image.svg` is properly handled and added to the build.

But, if I specify the same asset inside the template, like:

```
<template>

  <div>

    <img src="../images/image.svg" alt="">

  </div>

</template>
```

The build fails with "no transformer found".

**Reply by DeMoorJasper** [reply]
Strange, are you using the latest nightly release? If you are this is probably a bug and than I'll look into it

**Reply by msimpson** [reply]
Yes. I'm on `2.0.0-nightly.776`.
